.github/workflows/publish.yml

```diff
@@ -36,7 +38,16 @@ jobs:
         run: |
           git config --local user.email "Workflow: ${{ env.GITHUB_WORKFLOW }}[bot]"
           git config --local user.name "${{ env.GITHUB_WORKFLOW }}[bot]@workflow"
-          npm version ${{ github.event.inputs.segment }} -m "Publish v%s"
+          if [ "${{ github.event.inputs.segment}}" == pre* ]; then
+            PRE_ID_COMMAND_STR=""
+            PRE_ID="${{ github.event.inputs.preId}}"
+            if [ -n "$PRE_ID" ]; then
+                PRE_ID_COMMAND_STR="--preid $PRE_ID"
+            fi
+            npm version ${{ github.event.inputs.segment }} $PRE_ID_COMMAND_STR -m "Publish v%s"
```

Comment:
It's minor, but if you move like 42 up, then you can have just this one line of `npm version` rather than duplicating it.

Reply:
Ah true, good catch, this is the second iteration of the script and i totally missed that. I'll update it real